ml-agents/mlagents/trainers/models.py

```diff
@@ -407,7 +409,7 @@ def create_discrete_action_masking_layer(all_logits, action_masks, action_size):
             for i in range(len(action_size))
         ]
         raw_probs = [
-            tf.multiply(tf.nn.softmax(branches_logits[k]) + 1.0e-10, branch_masks[k])
+            tf.multiply(tf.nn.softmax(branches_logits[k]) + 1.0e-7, branch_masks[k])
```

Comment:
Can you explain the change from `1.0e-10` to `1.0e-7` here and below?

Reply:
In SAC I found that 1.0e-10 was too small to prevent NaNs. I'm changing it here to be safe, though I don't think there were any known NaNs for PPO.